server/data/apps.json

```
@@ -2804,5 +2804,40 @@
         "elementURL": "https://cast.garden/about/instance"
       }
     ]
+  },
+  {
+    "appName": "IPFS Podcasting",
+    "appType": [
+      "hosting",
```

The `directory` type might also apply since you are hosting a directory of IPFS-ified podcast feeds and episodes.

```suggestion
      "directory",
      "hosting",
```